### PR TITLE
Add documentations for generate_course_statistics and rank_courses

### PR DESCRIPTION
**Pull request by yhchen20**

### CODE REVIEW

File: R/frgtracker.R [hunk 1]
```diff
@@ -9,16 +9,47 @@
...
 
 # function4 start
 
+#' Calculate students' course grades to rank courses in ascending/descending order by a
+#' specified method.
+#'
+#' @param method The method applied to rank the courses, should be "mean", "1st-quantile",
+#' "median", or "3rd-quantile". Defaults to "mean"
+#' @param descending A logical value to decide if the rank should be in descending or
+#' ascending order.
```
Comment (yhchen20) [inline R/frgtracker.R]:
```suggestion
#' ascending order. Default to True
```

File: R/frgtracker.R [hunk 2]
```diff
@@ -9,16 +9,47 @@
...
+
 # function3 end
 
 # function4 start
 
+#' Calculate students' course grades to rank courses in ascending/descending order by a
+#' specified method.
+#'
+#' @param method The method applied to rank the courses, should be "mean", "1st-quantile",
+#' "median", or "3rd-quantile". Defaults to "mean"
```
Comment (jraza19) [inline R/frgtracker.R]: for the method - i would follow a slightly different syntax - I found this [example](https://www.rdocumentation.org/packages/DescTools/versions/0.99.40/topics/PostHocTest) from a function we are using in 554 lab - see the methods part.

Comment (yhchen20) [reply R/frgtracker.R]: Thanks for showing me the example, Javaria! I've changed the method part

**Review by jraza19 — APPROVED** [review]
LGTM! Great work :)

**Pull request merged**